support/image-builder/ami/create-ebs-image.sh

```diff
@@ -167,7 +167,7 @@ done
 sleep $SSH_WAIT
 
 while : ; do
- output=$(eval ssh -o StrictHostKeyChecking=no -i $KEY_FILE root@$IP 'tdnf install -y kpartx wget')
+ output=$(eval ssh -o StrictHostKeyChecking=no -i $KEY_FILE ec2-user@$IP 'sudo yum install -y kpartx wget')
```

Comment:
should it be tdnf instead of yum?

Reply:
Actually, I switched to using an Amazon Linux 2 image as the initial seed, since Photon OS was not available in all regions.